app/controllers/concerns/spotlight/catalog.rb

```diff
@@ -27,12 +27,7 @@ def add_facet_visibility_field
                                         }
     end
 
-    def render_save_this_search?
-      (current_exhibit && can?(:curate, current_exhibit)) &&
-        !(params[:controller] == 'spotlight/catalog' && params[:action] == 'admin')
-    end
-
-    def render_bulk_actions?
+    def render_curator_actions?
```

Comment:
I'm good with this, but @ggeisler is there a reason on when we would want to show "Save search" and "Bulk actions" separately? Does this make #2685 impossible?

Reply:
We should be able to put `if`/`unless` conditions on the individual tools.

Reply:
WFM